### PR TITLE
[v7r3] presentation for #5309 issue

### LINKED ISSUE #5309: Use Read the Docs Sphinx Theme for documentation
Can we consider using next sphinx extensions?

`sphinx-rtd-theme`
**[RTD Sphinx theme](https://sphinx-rtd-theme.readthedocs.io/en/latest/installing.html#installation)** already contain CSS(Sass) styles and designed for both desktop and mobile devices. IMHO this theme provides a more readable view, especially in Code Documentation.

`sphinx-panels`
**[sphinx-panels](https://sphinx-panels.readthedocs.io/en/latest/#installation)** support [Bootstrap 4](https://getbootstrap.com/docs/4.0/getting-started/introduction/) framework and provide elements like [panels](https://dirac--5313.org.readthedocs.build/en/5313/index.html#documentation-sources), [tabs](https://dirac--5313.org.readthedocs.build/en/5313/AdministratorGuide/ServerInstallations/InstallingDiracServer.html#installing-runit), [dropdown](https://dirac--5313.org.readthedocs.build/en/5313/AdministratorGuide/ServerInstallations/InstallingDiracServer.html#primary-server-installation), etc. 

You can try it using [build](https://dirac--5313.org.readthedocs.build/en/5313/) of #5313.

### PR DESCRIPTION
**Pull request by TaykYoku**
closes #5309 

This PR provides some implementation sphinx_rtd_theme and sphinx_panels for #5309.

Image icons was created with [Adobe Illustrator](https://www.adobe.com/products/illustrator.html).

BEGINRELEASENOTES

*Docs
CHANGE: replace the theme by the sphinx_rtd_theme

ENDRELEASENOTES

### CODE REVIEW

**Comment by fstagni** [comment]
IMHO this looks OK. Mine is not a technical review, purely a visual/feeling one. Just fix (or disable) the py2/pylint report.

File: docs/source/index.rst [hunk 1]
```diff
@@ -50,33 +34,40 @@ or in this `presentation <https://indico.cern.ch/event/505613/contributions/2227
...
-* :ref:`search`
+=============
+Documentation
+=============
+
+.. panels::
+  :card: shadow + text-center
+  :img-top-cls: p-5
+
+  :img-top: _static/read.png
```
Comment (andresailer) [inline docs/source/index.rst:45]: Are you really sure using these pictures like this is OK following the license of that website? I got totally lost in the legalese.

I also think keeping the text that explains what can be found in the different guides was useful

Comment (TaykYoku) [reply docs/source/index.rst:45]: The image is distributed with a free license:

Знімок екрана 2021-08-10 о 11 58 27

And https://profile.freepik.com/license/free say that:

>Where you can use Freepik content*:
Website

but

> What you CANNOT DO:
Sublicense, sell or rent any contents (or a modified version of them)

I'm not sure I understand all the nuances of use, but based on the fact that the GPL 3 license applies to `/docs` (where these icons will lie), and following the flow of thought in this [thread](https://opensource.stackexchange.com/questions/2653/including-resources-with-ambiguous-license-for-gpl-application), I think you're right. Placing these pictures in `/docs` seems to fall under the definition of "Sublicense".

Comment (TaykYoku) [reply docs/source/index.rst]: Made new images, now there is no need to explore the features of the license.

>I also think keeping the text that explains what can be found in the different guides was useful

If you are interested in my opinion, then these explanations do not provide interesting information but only complicate the interface. IMHO by the short name you can intuitively understand what's inside.

Comment (andresailer) [reply docs/source/index.rst]: Those new pictures look very nice and avoid any license issues :+1: 
Please squash your commits so the files also disappear from the history.

Regarding the little texts. I think having a clear statement that the user guide also contains client installation instructions was helpful, same for the administrator guide containing server instructions. While the search functionality can of course discover this with more ease having the hints prominently on the main page might make it easier to discover the information without resorting to intuition, which for people unfamiliar with DIRAC with not be too reliable. Maybe even adding a few keywords to the sections would increase the value. The few words also take up a lot less space than the images.

File: environment-py3.yml [hunk 2]
```diff
@@ -83,6 +83,8 @@ dependencies:
   - typing >=3.6.6
   - pyyaml
   - pip:
+    - git+https://github.com/executablebooks/sphinx-panels.git@v0.6.0
```
Comment (andresailer) [inline environment-py3.yml:86]: Why do these packages need to be installed from their git repos, and not from standard locations (pypi has both of them).
And why these specific versions?

Comment (TaykYoku) [reply environment-py3.yml:86]: > Why do these packages need to be installed from their git repos, and not from standard locations (pypi has both of them).

Apparently, this theme suits us and this PR can be considered for merging. Then if I understand correctly, these packages should be added to `DIRACOS2`.

> And why these specific versions?

I don't think we need to stick to versions of these packages. I just take the latest versions of the packages.

Comment (andresailer) [reply environment-py3.yml:86]:
```suggestion
    - sphinx-panels
```

Should be enough?  (or `>=0.6.0` if needed) Similar for sphinx_rtd_theme
(can this be in dependencies? @chrisburr ?)
As this environment-py3 file is used in readthedocs and the CI the packages also need to be here, but with as few version contraints as possible.

File: docs/source/index.rst (hunk 1 above)
Comment (TaykYoku) [reply docs/source/index.rst]: return guides explanation texts, done

**Review by fstagni — APPROVED** [review]

**Comment by andresailer** [comment]
Please squash the commits so that the icons from the other website are not in the history.

**Comment by TaykYoku** [comment]
squashed

**Comment by andresailer** [comment]
Thanks!

**Review by andresailer — APPROVED** [review]

**Review by fstagni — APPROVED** [review]

**Pull request merged**